src/stock/containers/wine-search/wine-search.container.ts

```diff
@@ -35,7 +35,7 @@ export class WineSearchContainer implements OnChanges {
     private clear$ = this.showResults$.filter(val => !val)
         .map(() => []);
 
-    private winesToShow$ = this.control.valueChanges
+    public winesToShow$ = this.control.valueChanges
```

Comment:
public members don't have the public keyword in our app (since they are public by default)